libs/designsystem/src/lib/components/checkbox/checkbox.component.spec.ts

```diff
@@ -14,17 +16,18 @@ const checkboxSizeSm = fatFingerSize;
 const checkboxSizeMd = size('xxxl');
 
 describe('CheckboxComponent', () => {
-  let spectator: Spectator<CheckboxComponent>;
-  let ionCheckbox: HTMLIonCheckboxElement;
-
-  const createHost = createHostFactory({
+  const createComponent = createComponentFactory({
     component: CheckboxComponent,
-    declarations: [IonCheckbox],
+    imports: [IonicModule.forRoot({ mode: 'ios', _testing: true })],
```

Comment:
Is this change relevant to the issue at hand? Do you need to import the entire Ionic module?

Reply:
The `ion-checkbox` was not being rendered during testing in Chrome, which caused tests to fail when relying on padding to maintain the height of the component. 

As the `radio`-component is similar and its slotted `ion-radio` is actually being rendered during testing in chrome; I had a look at what was done in `radio.component.spec.ts` and (perhaps a bit too) trustfully did it the same way - causing `ion-checkbox` to actually be rendered 😄